File: src/kyu_7/spacify/c/spacify_test.c
#include <criterion/criterion.h>
#include <stdlib.h>

void tester(const char *t_str, char *expected);

Test(Example_Tests, should_pass_these_two_tests)
{
    tester("hello world", "h e l l o   w o r l d");
    tester("12345", "1 2 3 4 5");
}

File: src/kyu_7/spacify/java/SpacifyTest.java
package kyu_7.spacify.java;
import org.junit.Test;

import static org.junit.Assert.*;

public class SpacifyTest {
    @Test
    public void basicTest() {
        assertEquals("h e l l o   w o r l d",Spacify.spacify("hello world"));
        assertEquals("1 2 3 4 5",Spacify.spacify("12345"));
    }
}